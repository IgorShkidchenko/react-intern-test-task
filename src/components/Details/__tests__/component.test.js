import React from "react";
import { shallow } from "enzyme";

import DetailsComponent from "../component";

describe("<DetailsComponent />", () => {
  it("renders correctly", () => {
    const location = { state: { repository: { fullname: "name", clone_url: "url" } } }
    const wrapper = shallow(<DetailsComponent location={location} />);

    expect(wrapper).toMatchSnapshot();
  });
});

