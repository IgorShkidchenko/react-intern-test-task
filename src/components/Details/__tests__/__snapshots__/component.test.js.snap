// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailsComponent /> renders correctly 1`] = `
<div>
  <p>
    <a
      href="url"
    >
       Clone
    </a>
  </p>
  <div
    className="tables"
  >
    <StatsComponent
      repository={
        Object {
          "clone_url": "url",
          "fullname": "name",
        }
      }
    />
    <Connect(LanguageTableContainer)
      repository={
        Object {
          "clone_url": "url",
          "fullname": "name",
        }
      }
    />
  </div>
  <Link
    to="/"
  >
    Back
  </Link>
</div>
`;
